configure.ac

```diff
@@ -125,6 +125,11 @@ AC_ARG_ENABLE([upnp-default],
   [use_upnp_default=$enableval],
   [use_upnp_default=no])
 
+AC_ARG_ENABLE([external-signer],
+    [AS_HELP_STRING([--disable-external-signer],[do not compile external signer support (default is to compile if Boost::Process is found)])],
```

Comment:
In commit "configure: add --enable-external-signer" (ec6af5d1c993d584861be4ce6a0f5188f1af73af)

Before we actually check in code making this flag useful, the `./configure --help` text should say `(incomplete, experimental)` or similar to avoid suggesting the build will support a feature that isn't actually usable.